Makefile

```diff
@@ -55,3 +55,33 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+DURATION ?= 3s
+COUNT ?= 5
+BENCH_PATH ?= bench
+TEST_PATH ?= ./...
```

Comment:
Move these variables to the top level, where other variables places.

Reply:
Done.